docs/integrations/skopt.rst

```diff
@@ -1,48 +1,103 @@
 .. _integrations-scikit-optimize:
 
-Neptune-Scikit-Optimize Integration
-===================================
+Neptune - Scikit-Optimize Integration
+=====================================
 
-This integration lets you monitor |Scikit-Optimize| (skopt) hyperparameter optimization in Neptune.
+|Run on Colab|
 
-.. image:: ../_static/images/integrations/skopt_neptuneai.png
-   :target: ../_static/images/integrations/skopt_neptuneai.png
-   :alt: Scikit Optimize Neptune integration
+What will you get with this integration?
+----------------------------------------
 
-Requirements
-------------
+|skopt-tour|
 
-To use Neptune + Scikit-Optimize integration you need to have installed is |neptune-client| and |neptune-contrib|.
+|Scikit-Optimize|, or skopt, is a simple and efficient library to minimize (very) expensive and noisy black-box functions. 
+With Neptune integration, you can:
 
-.. code-block:: bash
+- visualize the runs as they are running,
+- see charts of logged run scores,
+- log the parameters tried at every run,
+- log figures from plots module: plot_evaluations, plot_convergence, plot_objective, and plot_regret,
+- monitor hardware consumption during the run,
+- save the pickled results object.
+
+.. tip::
+    You can log many other experiment metadata like interactive charts, video, audio, and more.
+    See the :ref:`full list <what-you-can-log>`.
+	
+.. note::
+
+    This integration is tested with ``neptune-client==0.4.130``, ``neptune-contrib==0.25.0``, and ``scikit-optimize==0.8.1``
+	
+Where to start?
+---------------
+To get started with this integration, follow the :ref:`quickstart <skopt-quickstart>` below. 
+You can also skip the basics and take a look at how to change what you want to log after training in the :ref:`advanced options <skopt-advanced-options>` section.
+
+If you want to try things out and focus only on the code you can either:
+
+#. Open the Colab notebook (badge-link below) with quickstart code and run it as an anonymous user "`neptuner`" - zero setup, it just works,
+#. View quickstart code as a plain Python script on GitHub.
+
+|Run on Colab|
+
+.. _skopt-quickstart:
 
-    pip install neptune-client neptune-contrib['monitoring']
+Quickstart
+----------
+This quickstart will show you how to:
 
+* Install the necessary neptune packages
+* Connect Neptune to your Skopt hyperparameter tuning code and create the first experiment
+* Log metrics, figures, and artifacts from your tuning job to Neptune, and 
```

Comment:
I don't think we will be logging figures right? 
Also, are we logging the results.pkl objecT?

Reply:
We are logging figures under [diagnostics](https://ui.neptune.ai/shared/scikit-optimize-integration/e/SCIK-5/logs) and results.pkl under [artifacts](https://ui.neptune.ai/shared/scikit-optimize-integration/e/SCIK-5/artifacts)